src/annotator/test/guest-test.js

```diff
@@ -79,29 +78,21 @@ describe('Guest', () => {
     };
     CrossFrame = sandbox.stub().returns(fakeCrossFrame);
 
-    fakeHTMLIntegration = {
+    fakeIntegration = {
```

Comment:
Given that this is a relatively-complex fake, would there be any value in typing it to the `Integration` type? This might help catch oversights in how this is structured as things evolve. This isn't a pattern we've used in test fakes to this point (I don't think?).

Reply:
Currently tests are not type-checked, so it would only be information for the reader about intent. We might want to reconsider this in future.